File: Bahasa/app/src/main/java/ds/id/Bahasa/Controls/ScreenLockutil.java
package ds.id.Bahasa.Controls;

import android.view.Window;
import android.view.WindowManager;

public class ScreenLockutil {

    public static void Lock_Flag(Window window, boolean bScreenLock)
    {
        if(bScreenLock) {
            window.addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON);
        } else {
            window.clearFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON);
        }
    }

}

File: Bahasa(kotlin)/app/src/main/java/ds/id/Bahasa/Controls/ScreenLockutil.kt
package ds.id.Bahasa.Controls

import android.view.Window
import android.view.WindowManager

object ScreenLockutil {

    fun Lock_Flag(window: Window, bScreenLock: Boolean) {

        if (bScreenLock) {
            window.addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)
        } else {
            window.clearFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)
        }
    }

}